ios/DroidKaigi 2021/DroidKaigi 2021/GoogleService-Info-Stating-Debug.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file's name can be `GoogleService-Info.plist`.
Did there have some  problems?

Reply:
I thought it configured with multiple files.
(one for debug, one for release...).
Isn't it?

Reply:
I'm sorry. It's my misunderstanding 🙇🏼 

It could be done if the project source directories were separate, but this time it wasn't possible.